pages/rdm_cycle/sharing.md

```diff
@@ -23,35 +23,33 @@ Even though it may not be possible to openly share all data because of ethical,
 ## What should be considered for Data Sharing?
 
 * Take Data Sharing into consideration already at the [Planning stage](planning). Don't leave it to the last stage. 
+  - For collaborative projects use repositories and sharing services which allow controlled access to share your preliminary data with project partners. 
+  - Consider depositing your data to a public repository as early as possible. This saves a lot of trouble later on. Data can be put under embargo until you want to release it, e.g. at the time of article publication.
 * Making the data as FAIR as possible will ensure that the most value can be made out of it in the future.
   - The easiest way to make data FAIR is to deposit it in a publicly accessible recognised repository.
```

Comment:
I would maybe say "The easiest way to make data **more** FAIR is...", since publishing data might not be enough to have FAIR data.